WORKSPACE

```diff
@@ -52,3 +52,15 @@ bind(
     name = "gson",
     actual = "@gson_maven//jar",
 )
+
+http_archive(
+    name = "io_bazel",
+    urls = ["https://github.com/bazelbuild/bazel/archive/master.zip"],
```

Comment:
This looks wrong. We just need zlib. Why are we pulling in the whole bazel repo?